specification/resources/resource-manager/Microsoft.Resources/stable/2019-05-10/resources.json

```diff
@@ -155,12 +155,7 @@
             }
           }
         },
-        "x-ms-long-running-operation": true,
-        "x-ms-examples": {
-          "Create deployment at management group scope.": {
-            "$ref": "./examples/PutDeploymentAtManagementGroup.json"
```

Comment:
Why are you removing this example?

Reply:
Added back all examples in 2019-05-10. I assume there will be separate PR for other API versions. Thx!